src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,58 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private StringBuilder stringBuilder;
+    private String resultString;
+
+    private void readDataFromFile(String nameFile) {
+        File file = new File(nameFile);
+        try (BufferedReader reader = new BufferedReader(new FileReader(file))) {
+            stringBuilder = new StringBuilder();
+            String value = reader.readLine();
+            while (value != null) {
+                stringBuilder.append(value).append(System.lineSeparator());
+                value = reader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Cant read file", e);
+        }
+    }
+
+    private void processData() {
+        int sumSupply = 0;
+        int sumBuy = 0;
+        String[] split = stringBuilder.toString().split("\\W+");
+        for (int i = 0; i < split.length; i++) {
```

Comment:
it would be easier if you iterate through array of strings like ["supply,100", "buy,100"...]
Because here you have an array with mixed operations and values, any modification to such array will brake your logic